terra-base-jupyter/README.md

```diff
@@ -0,0 +1,91 @@
+# terra-base-jupyter image
+
+This repo contains the terra-base-jupyter image that is compatible with the
+jupyter notebook service in [Terra]("https://app.terra.bio/") called Leonardo.
+The goal is to provide a lean, GPU-enabled base image that can be use for both Terra-suported, and custom image builds.
+
+The goal is to provide the user with a base conda environment where they can install any
+package they need, while keeping the jupyter server itself in an isolated virtual environment
+that DSP devs control using poetry as the dependency manager
+
+## Image contents
+
+`terra-base-jupyter` extends an cuda 12.2.0 and Ubuntu 22.04 base image with the minimum 
+requirements necessary to set up Jupyter and provide compatibility with Leonardo and GPU machines.
+
+- OS prerequisites
+- Miniconda with python 3.10
+- Two python virtual environments based on miniconda python
+-- The user base Conda environment, full list of python packages is available [here](conda-environment.yml)
+-- The jupyter virtual environment managed separately by poetry, full list of python packages is available [here](pyproject.toml) that contains the terra-specific configurations (e.g. WelderContentManager)
+
+To see the complete contents of this image please see the [Dockerfile](./Dockerfile).
+
+## Contributing
+
+### Installing python 3.10
+
+All terra-docker images are on Python 3.10, so make sure to have it installed on your
+local machine, for mac users doing this via `pyenv` is the recommended way of doing it:
+
+```bash
+$> brew install pyenv
+$> pyenv install 3.10.0
+```
+
+### Installing poetry
+
+All python package dependencies are managed by [poetry]("https://python-poetry.org/docs/"),
+which allows for cross-platforms, deterministic python environments builds.
+
+You can install poetry via brew or curl commands (NEVER USE PIP!):
+
+```bash
+# Via brew
+$> brew install poetry
+# Via curl
+$> curl -sSL https://install.python-poetry.org | python3 -
+```
```

Comment:
... it may have changed since this writing, but `poetry`'s recommended install instructions are to use `pipx` now

Reply:
Actually it looks like the official installer is still curl? https://python-poetry.org/docs/#installing-with-the-official-installer